store/mongo/datastore_mongo.go

```diff
@@ -234,6 +235,36 @@ func (db *DataStoreMongo) UpdateLoginTs(ctx context.Context, id string) error {
 	return err
 }
 
+func (db *DataStoreMongo) GetUserForLogin(
+	ctx context.Context,
+	email string,
+	loginTS time.Time,
+) (*model.User, error) {
+	var user model.User
+
+	err := db.client.Database(mstore.DbFromContext(ctx, DbName)).
+		Collection(DbUsersColl).
+		FindOneAndUpdate(ctx,
```

Comment:
so with this change, mass login attempt will result in mass db write. what if I employ a farm of machines trying to brute force hM password?

Reply:
> so with this change, mass login attempt will result in mass db write. what if I employ a farm of machines trying to brute force hM password?

perhaps we should also get some reasonable local (as in: per useradm instance) rate limits?

Reply:
I think this should be the responsibility of the gateway (Traefik) to prevent abuse of endpoints like this.

Reply:
I would say that if we do not have anything like it in place, then we need a follow up ticket, opinion?